package.json

```diff
@@ -20,10 +20,12 @@
     "@react-navigation/bottom-tabs": "^5.1.1",
     "@react-navigation/native": "^5.0.9",
     "@react-navigation/stack": "^5.1.1",
+    "@types/yup": "^0.28.3",
```

Comment:
This should go to `devDependencies`, not `dependencies`.

Reply:
You right. I forgot that the dev dependencies are existing at all :smile: :+1: